Jenkinsfile_nightly

```diff
@@ -30,7 +36,9 @@ static Map<String, Object> secret(String secretName, String envVariable) {
 def secrets = [
   'fis-${env}'    :[
     secret('idam-ui-secret', 'IDAM_SECRET'),
-    secret('idam-system-user-password', 'TEST_PASSWORD')
+    secret('idam-system-user-password', 'TEST_PASSWORD'),
+    secret('citizen-user', 'CITIZEN_USERNAME'),
```

Comment:
do we have these secrets in azure? if yes, have you added them manually?

Reply:
Yes, it's same as we used for Functional testing and saved in Azure Vault